src/main.ts

```diff
@@ -16,7 +16,7 @@ async function run(): Promise<void> {
       })
     } else {
       core.info(`adding to path: ${tool}`)
-      chmodr(tool, 0o0755, err => {
+      chmod(tool, 0o0755, err => {
```

Comment:
Please note that `chmodr` used to apply the mode recursively.
This is not the case for `fs.chmod`.